### PR TITLE
made sign up and sign in responsive

### PR DESCRIPTION
**Pull request by Ralatcode**
Made sign up and sign in page mobile responsive, are we still leaving the google sigin in or it is getting removed?

### CODE REVIEW

File: runner/style.css [hunk 1]
```diff
@@ -323,3 +323,78 @@ html {
     text-decoration: none;
     color: var(--link-color);
 }
+
+/* mobile and tablet views */
+ @media (max-width: 850px) {
```
Comment (Edema-wxd) [inline runner/style.css:328]: please add 350 px for mobile view

Comment (Edema-wxd) [inline runner/style.css]: please add for mobile view, 380px

Comment (Ralatcode) [reply runner/style.css]: Adjusted

File: runner/style.css [hunk 2]
```diff
@@ -324,8 +324,8 @@ html {
     color: var(--link-color);
 }
 
-/* mobile and tablet views */
- @media (max-width: 850px) {
+/* mobile view */
```
Comment (Edema-wxd) [inline runner/style.css:327]: soft lets see

File: runner/style.css [hunk 3]
```diff
@@ -323,3 +323,78 @@ html {
     text-decoration: none;
     color: var(--link-color);
 }
+
+/* mobile view */
+ @media (max-width: 480px) {
```
Comment (Edema-wxd) [inline runner/style.css:328]: nicee

**Review by Edema-wxd — APPROVED** [review]

**Pull request merged**